models/pim_ext_p.go

```diff
@@ -0,0 +1,88 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/container"
+)
+
+const (
+	RnPimExtP        = "pimextp"
+	DnPimExtP        = "uni/tn-%s/out-%s/pimextp"
+	ParentDnPimExtP  = "uni/tn-%s/out-%s"
+	PimExtPClassName = "pimExtP"
+)
+
+type ExternalProfile struct {
```

Comment:
Will this not conflict with other external profiles? should we make it more specific?

Reply:
ok changed the name